src/comparable.ml

```diff
@@ -0,0 +1,13 @@
+module type S = sig
+  type t
+
+  include Equatable.S with type t := t
+
+  val compare:t Comparator.t
+end
+
+module Default(T: sig type t end) : S = struct
```

Comment:
Well, it is late in London now. Let me update this tomorrow, I have some idea we can try.